specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/computeRPCommon.json

```diff
@@ -0,0 +1,1916 @@
+          "enum": [
+            "ImageDefault",
+            "AutomaticByPlatform"
+          ],
+          "x-ms-enum": {
+            "name": "LinuxVMGuestPatchMode",
+            "modelAsString": true
+          }
+        },
+        "assessmentMode": {
+          "type": "string",
+          "description": "Specifies the mode of VM Guest Patch Assessment for the IaaS virtual machine.<br /><br /> Possible values are:<br /><br /> **ImageDefault** - You control the timing of patch assessments on a virtual machine. <br /><br /> **AutomaticByPlatform** - The platform will trigger periodic patch assessments. The property provisionVMAgent must be true.",
+          "enum": [
+            "ImageDefault",
+            "AutomaticByPlatform"
+          ],
+          "x-ms-enum": {
+            "name": "LinuxPatchAssessmentMode",
+            "modelAsString": true
+          }
+        },
+        "automaticByPlatformSettings": {
+          "$ref": "#/definitions/LinuxVMGuestPatchAutomaticByPlatformSettings",
+          "description": "Specifies additional settings for patch mode AutomaticByPlatform in VM Guest Patching on Linux."
+        }
+      },
+      "description": "Specifies settings related to VM Guest Patching on Linux."
+    },
+    "WindowsVMGuestPatchAutomaticByPlatformSettings": {
+      "type": "object",
+      "properties": {
+        "rebootSetting": {
+          "type": "string",
+          "description": "Specifies the reboot setting for all AutomaticByPlatform patch installation operations.",
+          "enum": [
+            "Unknown",
+            "IfRequired",
+            "Never",
+            "Always"
+          ],
+          "x-ms-enum": {
+            "name": "WindowsVMGuestPatchAutomaticByPlatformRebootSetting",
+            "modelAsString": true
+          }
+        },
+        "bypassPlatformSafetyChecksOnUserSchedule": {
+          "type": "boolean",
+          "description": "Enables customer to schedule patching without accidental upgrades"
+        }
+      },
+      "description": "Specifies additional settings to be applied when patch mode AutomaticByPlatform is selected in Windows patch settings."
+    },
+    "LinuxVMGuestPatchAutomaticByPlatformSettings": {
+      "type": "object",
+      "properties": {
+        "rebootSetting": {
+          "type": "string",
+          "description": "Specifies the reboot setting for all AutomaticByPlatform patch installation operations.",
+          "enum": [
+            "Unknown",
+            "IfRequired",
+            "Never",
+            "Always"
+          ],
+          "x-ms-enum": {
+            "name": "LinuxVMGuestPatchAutomaticByPlatformRebootSetting",
+            "modelAsString": true
+          }
+        },
+        "bypassPlatformSafetyChecksOnUserSchedule": {
```

Comment:
>bypassPlatformSafetyChecksOnUserSchedule [](http://example.com/codeflow?start=9&length=40)

Consider if you can use enum over boolean, and also LinuxVMGuestPatchAutomaticByPlatformSettings and WindowsVMGuestPatchAutomaticByPlatformSettings are very similar, what's the difference between them?

Reply:
The rest looks good, see if this can be updated